static/randomizer.js

```diff
@@ -0,0 +1,35 @@
+
+(function (){
```

Comment:
For this purpose wouldn't it be simpler and better to implement only function instead of class? We can later change it to class if we need it.